modules/service/src/main/resources/db/migration/V0371__target_role.sql

```diff
@@ -0,0 +1,66 @@
+create type e_target_role as enum('science', 'guide');
+comment on type e_existence is 'The role of the target.';
+
+alter table t_target
+add column c_role e_target_role;
```

Comment:
Seems like this should be `not null` with a clause to set all existing targets to `science`.

Reply:
Oh, good idea. If I knew you could do that, I had forgotten.

Reply:
I still don't know how to do this. All examples I can find on the web show adding the columns as nullable, updating the data, then setting the not null constraint. Do you have an example for this?

Reply:
Yep, I thought you could use a `USING` clause but that only works when modifying an existing column. What you have here is fine.